go/test/endtoend/cluster/vtorc_process.go

```diff
@@ -60,7 +61,7 @@ func (orc *VtorcProcess) Setup() (err error) {
 	orc.proc.Args = append(orc.proc.Args, orc.ExtraArgs...)
 	orc.proc.Args = append(orc.proc.Args, "-alsologtostderr", "http")
 
-	errFile, _ := os.Create(path.Join(orc.LogDir, "orc-stderr.txt"))
+	errFile, _ := os.Create(path.Join(orc.LogDir, "orc-stderr.txt"+time.Now().String()))
```

Comment:
- nit: keep `.txt` as file extension. Inject the timestamp before the extension.
- nit: `time.Now().String()` has spaces. Perhaps use unix time converted to string, which is only numeric.

Reply:
Changed